### PR TITLE
enh(axapta) Update X++ keyword list

### LINKED ISSUE #2686: (axapta) Typo in keyword name; keyword list needs updating
The Axapta (X++) keyword list is out of date and needs updating. Also the display keyword has a typo, disaply

### PR DESCRIPTION
**Pull request by ryanjonasson**
Closes #2686

### CODE REVIEW

**Comment by joshgoebel** [comment]
Nice.

This is a lot of new keywords, is this list documented somewhere?

**Comment by ryanjonasson** [comment]
Not to my knowledge. This is sourced from the X++ Compiler team at Microsoft with their permission.

**Comment by joshgoebel** [comment]
Looks good (hard to review with no canonical source)...

Could we please:

- Refactor to "one keyword per line" (see the recent changes to sharp, etc) for make future maintenance easier.
- Add a entry to CHANGES.md

Thanks,
Josh

**Comment by joshgoebel** [comment]
Is X++ what they call it now?  Any chance you know enough to split those into built in vs literals vs keywords?  If not we can merge as-is.

**Comment by ryanjonasson** [comment]
The language has always been called X++, the product it is used in used to be called Axapta, and the two became synonymous.
I'll take a swing at breaking the list apart.

**Comment by joshgoebel** [comment]
We can't rename the file easily but I'm open to updating the name metadata and our docs if there is a better name to use for this now (that's broadly recognizable).

Currently we have: "Microsoft Axapta (now Dynamics 365)" with no mention of X++ anywhere.

File: src/languages/axapta.js [hunk 1]
```diff
@@ -1,20 +1,152 @@
...
+  ];
+
+  var KEYWORDS = {
+    keyword: NORMAL_KEYWORDS.join(' '),
+    built_in: BUILT_IN_KEYWORDS.join(' '),
+    literal: LITERAL_KEYWORDS.join(' ')
+  };
+
   return {
     name: 'Dynamics 365',
```
Comment (joshgoebel) [inline src/languages/axapta.js:148]: But is it _'now'_ "Microsoft Dynamics 365" or is it X++ for naming purposes?

Comment (ryanjonasson) [reply src/languages/axapta.js:148]: The language is X++, exclusively used in Dynamics 365. So how about 
Language: Microsoft X++
Description: X++ is a language used in Microsoft Dynamics 365, Dynamics AX, and Axapta.

Comment (ryanjonasson) [reply src/languages/axapta.js:148]: And would update name to be 'X++'

Comment (joshgoebel) [reply src/languages/axapta.js:148]: Perhaps update the link to

https://docs.microsoft.com/en-us/dynamicsax-2012/developer/x-language-programming-guide

?

File: src/languages/axapta.js [hunk 2]
```diff
@@ -1,20 +1,154 @@
...
+    'utcdatetime',
+    'var'
+  ];
+
+  const LITERAL_KEYWORDS = [
+    'default',
+    'false',
+    'null',
+    'true',
+
```
Comment (joshgoebel) [inline src/languages/axapta.js:36]:
```suggestion
```

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@ryanjonasson Thanks for all the work on this!

**Pull request merged**